engine/access/rpc/backend/backend_test.go

```diff
@@ -238,6 +246,18 @@ func (suite *Suite) TestTransactionStatusTransition() {
 	txID := transactionBody.ID()
 	blockID := block.ID()
 
+	ids := unittest.IdentityListFixture(1)
+	receipt := unittest.ReceiptForBlockFixture(&block)
+	suite.receipts.
+		On("ByBlockIDAllExecutionReceipts", mock.Anything).
+		Return([]flow.ExecutionReceipt{*receipt}, nil)
+	receipt.ExecutorID = ids[0].NodeID
```

Comment:
Should we do this before the mocking call? On line 253 we're copying the receipt, I don't think this field change will be present when `ByBlockIDAllExecutionReceipts` is called.